instructions/build_instructions_windows.md

```diff
@@ -15,14 +15,15 @@ Default install paths are assumed in all projects and scripts, be prepared to ad
 * Clone VSRepo into the VapourSynth dir (`git clone https://github.com/vapoursynth/vsrepo`)
 * Clone zimg v2.9 branch into the VapourSynth dir (`git clone https://github.com/sekrit-twc/zimg --branch v2.9`)
 * Clone avs+ pinterf mt branch into the VapourSynth dir (`git clone https://github.com/pinterf/AviSynthPlus --branch MT`)
+* Set environment variables `PYTHON32` and `PYTHON64` for the msvc project and cython build
```

Comment:
> Set `PYTHON32` and `PYTHON64` for the msvc project.

The Cython-Build shouldn't need those variables. It just takes the python-interpreter that actually runs `python.exe`

Reply:
From previous version [both architectures were used](https://github.com/vapoursynth/vapoursynth/blob/master/cython_build.bat#L4-L7). My version [builds separated](https://github.com/Youka/vapoursynth/blob/master/cython_build.bat#L6-L13) (one architecture per run) but still preserves this behaviour.

If this was already wrong before, this can be fixed by another commit.

Reply:
Perhaps we should just delete the `cython_build.bat`. All windows versions install the `py`-launcher.

Running `py -3.7-32 setup.py build_ext` and `py -3.7 setup.py build_ext` should be easy enough. And those wo have multiple competing python installations of the same version and architecture should know how call them directly.

Reply:
Not necessarily the `py` launcher is always installed. Portable python versions exist too (as used by AppVeyor f.e.) and should be supported for flexibility. I left both calls, x86 and x64, in `cython_build.bat` but could call `python` also (because set to PATH for the current run).

Reply:
Still, in those cases, the developer *should* know how to run the respective python-version directly.

Thus it is not neccessary to use those batch-files imo.

Reply:
Would be more windows instruction entries or expectations that users are already experts.

Let's solve such minor problems later, not delaying this PR for other things than his purpose.